subsys/llext/llext.c

```diff
@@ -221,3 +221,37 @@ int llext_call_fn(struct llext *ext, const char *sym_name)
 
 	return 0;
 }
+
+void llext_bootstrap(struct llext *ext, llext_entry_fn_t entry_fn, void *user_data)
+{
+	int ret = llext_get_init_array(ext, NULL, 0);
+
+	if (ret < 0) {
+		LOG_ERR("Failed to get init array count");
+		return;
+	}
+
+	if (ret > 0) {
+		typedef void (*elf_init_fn_t)(void);
+
+		int init_fn_count = ret / sizeof(elf_init_fn_t);
+		elf_init_fn_t init_fns[init_fn_count];
```

Comment:
should we check `ret` for a maximum value before allocating an arbitrary size array on stack?